File: bitget-php-sdk-api/src/model/ws/WsLoginReq.php
<?php


namespace bitget\model\ws;


class WsLoginReq
{
    var string $apiKey;
    var string $passphrase;
    var string $timestamp;
    var string $sign;

    /**
     * WsLoginReq constructor.
     * @param string $apiKey
     * @param string $passphrase
     * @param string $timestamp
     * @param string $sign
     */
    public function __construct(string $apiKey, string $passphrase, string $timestamp, string $sign)
    {
        $this->apiKey = $apiKey;
        $this->passphrase = $passphrase;
        $this->timestamp = $timestamp;
        $this->sign = $sign;
    }


}

File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/ws/WsLoginReq.java
package com.bitget.openapi.dto.request.ws;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WsLoginReq {
    private String apiKey;
    private String passphrase;
    private String timestamp;
    private String sign;
}
